packages/esm-ward-app/src/ward-patient-card/ward-patient-card.tsx

```diff
@@ -1,25 +1,42 @@
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
 import { type WardPatientCardProps } from '../types';
 import { usePatientCardRows } from './ward-patient-card-row.resources';
 import styles from './ward-patient-card.scss';
-import { getPatientName, launchWorkspace } from '@openmrs/esm-framework';
-import { type WardPatientWorkspaceProps } from '../ward-patient-workspace/ward-patient.workspace';
+import { getPatientName, launchWorkspace, closeWorkspace } from '@openmrs/esm-framework';
+import { type ActiveBedSelection, getWardStore, type WardStoreState } from '../store';
+import classNames from 'classnames';
 
 const WardPatientCard: React.FC<WardPatientCardProps> = (props) => {
+  const wardStore = getWardStore();
   const { locationUuid } = useParams();
   const patientCardRows = usePatientCardRows(locationUuid);
+  const [activeBedSelection, setActiveBedSelection] = useState<ActiveBedSelection | null>(null);
+
+  const updateActiveBedSelection = useCallback((state: WardStoreState) => {
```

Comment:
this is the functionality that updates with bed is active/highlighted?  I would call it updateSelectedBed or updateActiveBed... "activeBedSelection" is a bit redundant and confusing.

Also, just to clarify, what happens in the cases where their are two patients in a bed?

Reply:
> what happens in the cases where their are two patients in a bed?
The workspace opens for a single patient. So the expected is that when a patient card is selected it sets the store with the `patient`, `bed` and `visit` details.

> I would call it updateSelectedBed or updateActiveBed... "activeBedSelection" is a bit redundant and confusing.
On this let me work on a relative variable name

Thanks @mogoodrich